apps/server/src/app.ts

```diff
@@ -324,3 +330,46 @@ process.on('uncaughtException', async (error) => {
 process.once('SIGHUP', async () => shutdown(0));
 process.once('SIGINT', async () => shutdown(0));
 process.once('SIGTERM', async () => shutdown(0));
+
+/**
+ * @description tries to open the server with the desired port, and if getting a `EADDRINUSE` will change to an efemeral port
```

Comment:
```suggestion
 * @description tries to open the server with the desired port, and if getting a `EADDRINUSE` will change to an ephemeral port
```

Questions: 
- what do we mean by ephemeral port? The app will move to a new port for its entire session. 
- would we want to persist the changes of the port?

Reply:
as far as i understand that is the name of the range of random ports provided by the OS - [Ephemeral port](https://en.wikipedia.org/wiki/Ephemeral_port)
 a better message is welcome

as written now it is saved to the settings... I think we talked about that both approaches has upsides, now that it only targets electron I think it leans a bit more to the, "it should be saved" sider

Reply:
I see, this makes sense. 
From the naming, I would assume that the ports are short lived, which may not be true since a) they last the whole session and b) we may even be saving it